### ISSUE TITLE
Aeon: input label

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#89]
Use the WAVE accessibility browser plugin to identify the issue within the page, if necessary.

Issue: 
```
An <input> (except types of image, submit, reset, button, or hidden), <select>, or <textarea> does not have a properly associated label. A properly associated label is:

-    a non-hidden <label> element with a for attribute value that is equal to the id of a unique form control
-    a <label> element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute
-    a non-empty title attribute, or
-    a non-empty aria-labelledby attribute
```

```
An <input> (except types of image, submit, reset, button, or hidden), <textarea>, or <select> element has a non-empty title attribute value and is missing a label or valid aria-labelledby reference.
```

Page found: 
- [Finding Aid Request](https://aeon.library.ucsc.edu/aeon/aeon.dll?Action=10&Form=31&Value=https%3A%2F%2Fcinco-prd.s3.amazonaws.com%2Fmedia%2Fead%2Fms339_1t64un6.xml) - missing label
- [Search](https://aeon.library.ucsc.edu/aeon/aeon.dll?Action=10&Form=2&Value=ViewSearchResults) - non-empty title, replace with label

To complete:
- [x] Inputs on the Request page have a properly associated label
- [x] Inputs on Search page replace title attribute with label
- [x] Spot check the code for other instances of this issue
- [x] Fix any other instances found

**Comment by rschwab** [comment]
The search page isn't showing this error anymore.

The request page will be tested Monday. There is one other checkbox in the aeon.xslt file for "noInventory" ([line 160](https://github.com/AtlasSystems/hosting-aeon-ucsc/blob/593978bcd6dd4e54aebd3029cd74443845f699b0/Aeon/aeon.xslt#L160)), but I haven't yet found where that is in the website to be able to test it.

**Comment by rschwab** [comment]
Errors are gone in testweb:

<img width="1521" height="894" alt="Image" src="https://github.com/user-attachments/assets/17700814-14d1-4d85-93fa-73503bdaa614" />
